Rakefile

```diff
@@ -409,11 +400,6 @@ task :check_manifest do
 end
 
 namespace :bundler do
-  desc "Initialize bundler submodule"
-  task :checkout do
-    sh "git submodule update --init"
```

Comment:
This now would have to be documented and explicitly called whenever a new developer would want to contribute to Rubygems?

Reply:
Ooops, I thought `git clone` would initialize the submodule by default, but it doesn't. Let me restore this.